portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm

```diff
@@ -55,6 +55,19 @@
     </li>
     #end
 
+  #if ($quickLinks)
+    <li>
+      <button class="btn icon-button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#sakai-quickLinksPanel"
+          aria-controls="sakai-quickLinksPanel"
+          aria-label="$rloader.sit_quick_links"
+          title="$rloader.sit_quick_links">
+        <i class="bi-link-45deg"></i>
```

Comment:
I would add an entry to library/src/skins/default/src/sass/base/_icons.scss to decouple this.  Add line under locked in that file.

Reply:
alright :)